src/wallet/rpcwallet.cpp

```diff
@@ -4051,8 +4051,8 @@ static const CRPCCommand commands[] =
     { "wallet",             "exportsaplingviewingkey",  &exportsaplingviewingkey,  true  },
     { "wallet",             "getshieldedbalance",       &getshieldedbalance,       false },
     { "wallet",             "listshieldedunspent",      &listshieldedunspent,      false },
-    { "wallet",             "raw_shielded_sendmany",    &raw_shielded_sendmany,    false },
-    { "wallet",             "shielded_sendmany",        &shielded_sendmany,        false },
+    { "wallet",             "rawshieldedsendmany",    &rawshieldedsendmany,    false },
+    { "wallet",             "shieldedsendmany",        &shieldedsendmany,        false },
```

Comment:
indentation

Reply:
added a 3rd commit here that unifies this section's alignment (independently of the others)